README.md

```diff
@@ -44,6 +44,7 @@ And that's all, you can connect to your embedded-elastic instance on specified p
 | `withSetting(String key, Object value)` | setting name and value as in elasticsearch.yml file |
 | `withPlugin(String expression)` | plugin that should be installed into Elasticsearch; treat expression as argument to `./elasticsearch-plugin install <expression>` command; use multiple times for multiple plugins |
 | `withIndex(String indexName, IndexSettings indexSettings)` | specify index that should be created and managed by EmbeddedElastic |
+| `withTemplate(String templateName, String templateBody)` | specify a template that should be created and managed by EmbeddedElastic |
```

Comment:
In `IndexBuilder` class we have methods `withSettings` and `withType` and both of them have two variants: one that takes String and one that takes InputStream. We should have same here

Reply:
Done